src/annotator/anchoring/range.js

```diff
@@ -249,7 +249,7 @@ export class NormalizedRange {
       } else {
         origParent = $(node).parent();
       }
-      const xpath = xpathFromNode(origParent, root)[0];
+      const xpath = xpathFromNode(origParent[0], root ?? document);
```

Comment:
The `root` argument to `xpathFromNode` is not optional. This keeps the tests passing, although I think the `root` argument here should really be required, in which case this fallback can be removed.